File: lib_common/src/main/java/com/wujia/lib_common/base/BaseView.java
package com.wujia.lib_common.base;

import android.content.Context;

/**
 * Created by xmren on 2018/5/4.
 */

public interface BaseView {
    void showErrorMsg(String msg);

    void showLoadingDialog(String text);

    void hideLoadingDialog();

    Context getContext();

    void onLoginStatusError();
}

File: lib_common/src/main/java/com/wujia/lib_common/base/BaseView.kt
package com.wujia.lib_common.base

/**
 * Created by xmren on 2018/5/4.
 */

interface BaseView {

//    val context: Context
    fun showErrorMsg(msg: String)

    fun showLoadingDialog(text: String)

    fun hideLoadingDialog()

    fun onLoginStatusError()
}
